packcheck.sh

```diff
@@ -1592,6 +1644,9 @@ build_compile () {
   show_step "Effective build config"
   show_build_config
 
+  # --------------------Git clone & merge-------------------
+  try_git_clone_and_merge "$GIT_SOURCE" "$GIT_REV" "$GIT_MERGE"
```

Comment:
We may want to do some sanity checks e.g. GIT_REV/GIT_MERGE makes sense only if GIT_SOURCE is specified.

Reply:
If `GIT_SOURCE` isn't specified `GIT_REV` & `GIT_MERGE` are not used. Should we error out or print a warning?